Code/lib/motor/motor.cpp

```diff
@@ -0,0 +1,91 @@
+#include "Arduino.h"
```

Comment:
Do you need to add the arduino header file here?

Reply:
move it to header file

Reply:
Yes. Because that is where I am using arduino functions

Reply:
Just for uniformity because you're including the header file, it is ok to include Arduino in the header file.